src/app/api/settings/route.ts

```diff
@@ -0,0 +1,46 @@
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+export async function POST(request: any) { 
```

Comment:
We are updating the smtp settings, using /api/settings route seems too generic, maybe you can try: /api/settings/smtp